common/static/common/js/components/views/feedback.js

```diff
@@ -0,0 +1,151 @@
+;(function (define) {
+    'use strict';
+    define(["jquery", "underscore", "underscore.string", "backbone", "common/js/components/utils/templates",
```

Comment:
Do remember to move this file, rather than creating a new copy of it. The Git history is valuable.